### PR TITLE
Auto shooting

### PR DESCRIPTION
**Pull request by BineGrid**

### CODE REVIEW

File: src/main/deploy/robotConfigs/TheDroidYoureLookingFor/sensors.conf [hunk 1]
```diff
@@ -1,13 +1,13 @@
 sensors {
     # Not using the ball feeder sensor for week 0 so it has x in front of name
```
Comment (Dev-Osmium) [inline src/main/deploy/robotConfigs/TheDroidYoureLookingFor/sensors.conf]: useless comment

File: src/main/java/frc/robot/commands/DefaultFeederCommand.java [hunk 2]
```diff
@@ -13,12 +13,23 @@
...
 import frc.robot.telemetries.Trace;
 
 public class DefaultFeederCommand extends CommandBase {
 
   BallFeederSensorBase m_feederSensor;
-  FeederBase m_feeder;
+  private FeederBase m_feeder;
+  private FeederStates m_feederState;
+  private static final double DEFAULT_STAGES_ONE_AND_TWO_SPEED = 0.3;
+  private static final double DEFAULT_DIFFERENCE_STAGE_TWO_AND_THREE_SPEED = 0.0;
```
Comment (Dev-Osmium) [inline src/main/java/frc/robot/commands/DefaultFeederCommand.java]: I spy magic numbers

File: src/main/java/frc/robot/commands/DefaultFeederCommand.java [hunk 3]
```diff
@@ -13,12 +13,23 @@
...
+  private static final double DEFAULT_STAGE_THREE_SPEED = DEFAULT_STAGES_ONE_AND_TWO_SPEED
+      + DEFAULT_DIFFERENCE_STAGE_TWO_AND_THREE_SPEED;
+  private static final double STAGE_TWO_SLOW_SPEED = 0.15;
```
Comment (Dev-Osmium) [inline src/main/java/frc/robot/commands/DefaultFeederCommand.java]: I spy more magic numbers

File: src/main/java/frc/robot/commands/FeedBothStagesIntoShooter.java [hunk 4]
```diff
@@ -48,7 +48,7 @@ public void initialize() {
   public void execute() {
     boolean shooterIsReady = m_shooterIsReady.getAsBoolean();
     if (shooterIsReady && (counter > 20)) {
-      m_feederBase.driveBothStages();
+      m_feederBase.runBothStages(0.6, 1.0);
```
Comment (EthanM827) [inline src/main/java/frc/robot/commands/FeedBothStagesIntoShooter.java]: magic numbers

File: src/main/java/frc/robot/groupcommands/parallelgroup/ShootWithRPM.java [hunk 5]
```diff
@@ -54,11 +56,20 @@ public ShootWithRPM(ShooterBase shooter, FeederBase feeder, double rpm) {
...
+    super.initialize();
+    m_isDone = false;
+  }
+
   @Override
   public void execute() {
     super.execute();
+    int numOfCells = m_ballFeederSensor.getNumberOfPowerCellsInFeeder();
+
+    System.out.println("Number of Powercells: " + numOfCells);
```
Comment (Dev-Osmium) [inline src/main/java/frc/robot/groupcommands/parallelgroup/ShootWithRPM.java]: Println in execute

**Comment by EthanM827** [comment]
Test on all 3 robots please

File: src/main/java/frc/robot/commands/FeedBothStagesIntoShooter.java [hunk 6]
```diff
@@ -23,6 +23,8 @@
   BooleanSupplier m_endCondition;
   BooleanSupplier m_shooterIsReady;
   double counter = 0;
+  private static final double kStageOneAndTwoSpeed = 0.6;
+  private static final double kStageThreeSpeed = 1.0;
```
Comment (Dev-Osmium) [inline src/main/java/frc/robot/commands/FeedBothStagesIntoShooter.java]: *sigh* alright

**Pull request merged**